packages/slice-machine/pages/slices.tsx

```diff
@@ -137,6 +137,7 @@ const SlicesIndex: React.FunctionComponent = () => {
                           .
                         </>
                       }
+                      data-testid="blank-slate"
```

Comment:
```suggestion
                      data-testid="slices-table-blank-slate"
```

I would suggest something specific to slices here, it's a specific blank test, and we should try to keep test id unique across the app on things that are unique.

Reply:
Perfect, good idea.